projects/spectator/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@ngneat/spectator",
```

Comment:
Please revert this and add the right peer dependency for jest.

Reply:
I added the peer dependency. Not sure what you mean by revert that though.  I didn't change the name.

diff --git a/projects/spectator/package.json b/projects/spectator/package.json
index 5c2c9d2..1cfe23e 100644
--- a/projects/spectator/package.json
+++ b/projects/spectator/package.json
@@ -2,7 +2,7 @@
   "name": "@ngneat/spectator",
   "description": "A powerful tool to simplify your Angular tests",
   "author": "Netanel Basal <netanel7799s@gmail.com>",
-  "version": "11.0.1",
+  "version": "11.0.2",
   "license": "MIT",
   "repository": {
     "type": "git",
@@ -31,7 +31,8 @@
   "peerDependencies": {
     "@angular/common": ">= 14.0.0",
     "@angular/router": ">= 14.0.0",
-    "@angular/animations": ">= 14.0.0"
+    "@angular/animations": ">= 14.0.0",
+    "jest": ">= 25.5.0"
   },
   "schematics": "./schematics/collection.json"
 }

Reply:
I mean revert `  "version": "11.0.2",`

Reply:
Done